### ISSUE TITLE
underline the text button.

### ISSUE THREAD

**Issue by dare-devilll** [Susmita-Dey/TextUtils#14]

**Comment by Susmita-Dey** [comment]
So, basically there will be a button which will underline the whole input text. Am I right?

**Comment by dare-devilll** [comment]
Yes.

**Comment by Susmita-Dey** [comment]
@dare-devilll I don't think it would be of any use. But links can be underlined after extracting or before extracting links.

**Comment by dare-devilll** [comment]
Yes, that sounds much better.

**Comment by Susmita-Dey** [comment]
@dare-devilll If you can do that, then change the issue title and let me know.

**Comment by fykaa** [comment]
Assign me :)

**Comment by Cypher1002** [comment]
i can do it can it be assigned to me under hacktoberfest
